test/setup_bundler_test.rb

```diff
@@ -6,18 +6,28 @@
 
 class SetupBundlerTest < Minitest::Test
   def test_does_nothing_when_running_in_the_ruby_lsp
-    Object.any_instance.expects(:system).with(bundle_install_command(update: false))
+    Object.any_instance.expects(:system).with(bundle_install_command)
     run_script("/some/path/ruby-lsp")
     refute_path_exists(".ruby-lsp")
   end
 
   def test_does_nothing_if_both_ruby_lsp_and_debug_are_in_the_bundle
-    Object.any_instance.expects(:system).with(bundle_install_command(update: false))
+    Object.any_instance.expects(:system).with(bundle_install_command)
     Bundler::LockfileParser.any_instance.expects(:dependencies).returns({ "ruby-lsp" => true, "debug" => true })
     run_script
     refute_path_exists(".ruby-lsp")
   end
 
+  def test_removes_ruby_lsp_folder_if_both_gems_were_added_to_the_bundle
+    Object.any_instance.expects(:system).with(bundle_install_command)
```

Comment:
Not related to this PR, but I feel we could use `Kernel.system` instead and make these mocks simpler:

```rb
Kernel.expects(:system)
```

Reply:
I actually tried that in the beginning, but for some reason it doesn't work. Not sure why.